pkg/phases/nginx/install.go

```diff
@@ -133,6 +133,12 @@ func Install(platform *platform.Platform) error {
 	if err := platform.ApplySpecs(v1.NamespaceAll, "nginx.yaml"); err != nil {
 		return err
 	}
+	// nginx alerts live in a separate manifest as alert syntax often cannot be templated
+	if !platform.Monitoring.IsDisabled() {
```

Comment:
but in this case it isn't an issue ?

Reply:
Not in this case no, but if we ever add an alert that needs a filter then it would be, so I figured it would be best to start with everything in the state that can handle it, rather than needing it to be reworked later.